### ISSUE TITLE
Installation instructions of dependencies should be updated

### ISSUE THREAD

**Issue by mdozmorov** [HajkD/LTRpred#13]
The LTRpred package requires six command-line tools, https://hajkd.github.io/LTRpred/articles/Introduction.html#installation. The installation instructions need to be updated as they refer to tools that are 2-3 versions older than the current ones. Consequently, installation instructions differ. For example:
````
wget ftp://selab.janelia.org/pub/software/hmmer3/3.1b2/hmmer-3.1b2.tar.gz
````
doesn't work. Only the latest version can be downloaded:
````
wget http://eddylab.org/software/hmmer/hmmer-3.3.tar.gz
````

Same for GenomeTools and others - installation instructions should be updated.

**Comment by HajkD** [comment]
Dear Mikhail,

Thank you for pointing this out. 

Unfortunately, there are several command-line tool dependencies that don't work with
the the latest versions in each case.

In the next issue you suggested to build a Docker container. I think this is an excellent idea and I will start working on this. 

Do you think a conda environment would also be useful?

Many thanks,
Hajk

**Comment by HajkD** [comment]
Dear Mikhail,

I now updated the installation instructions for the Linux tools using the most recent tool versions:
https://hajkd.github.io/LTRpred/articles/Introduction.html#install-tool-dependencies-on-linux

I decided to make these instructions only available for `Linux` since I am referring `macOS` and `Windows` users to the [ltrpred Docker](https://hub.docker.com/repository/docker/drostlab/ltrpred/) container.

Once more thank you very much for making me aware of this issue of the tool versions and I will make sure to keep the tools up to date in the future.

Many thanks,
Hajk

**Comment by mdozmorov** [comment]
Great work, and the software works now. Please, fix minor instruction issues, so we can close this issue:

- Add some readme to https://hub.docker.com/r/drostlab/ltrpred and its Rstudio variant
- Suggesting simplifying and streamlining Docker instructions, like:
    - Explicitly mention that the command-line version is suited for Windows, where port bridge cannot be made, so not RStudio access
    - Use -v volume mapping for the command-line version - the same type of analysis should be possible via command-line and RStudio versions. Terminal in RStudio is helpful
    - HMMER already installed within both images, so no need to install it within the containers
    - Explain why Dfam is not included (for one, it is very large)
    - Some minor typo fixes: "The rationalE for", "rertrotransposon",  "yeasT_genome"

**Comment by mdozmorov** [comment]
Also, recommend mentioning Docker options directly in the Git repo README. Many will want to use it

**Comment by HajkD** [comment]
Dear Mikhail,

I am very happy to hear that both containers are running on your machine.

Thank you for spotting the typos and for your suggestions to improve the documentation.

As suggested, I now added README files to https://hub.docker.com/r/drostlab/ltrpred and https://hub.docker.com/r/drostlab/ltrpred_rstudio .

In addition, I addressed all your points in the Vignette and pushed the updates to the repository:

- Explicitly mention that the command-line version is suited for Windows, where port bridge cannot be made, so not RStudio access
- Use -v volume mapping for the command-line version - the same type of analysis should be possible via command-line and RStudio versions. Terminal in RStudio is helpful
- HMMER already installed within both images, so no need to install it within the containers
- Explain why Dfam is not included (for one, it is very large)
- Some minor typo fixes: "The rationalE for", "rertrotransposon", "yeasT_genome"

Thank you very much for your help with this!

Best wishes,
Hajk
